src/client/auth-client.ts

```diff
@@ -33,6 +35,7 @@ const NANOSECONDS_PER_HOUR = NANOSECONDS_PER_SECOND * SECONDS_PER_HOUR;
 
 const IDENTITY_PROVIDER_DEFAULT = 'https://identity.internetcomputer.org';
 const IDENTITY_PROVIDER_ENDPOINT = '#authorize';
+const IDENTITY_PROVIDER_ICRC29_PATH = '/authorize';
```

Comment:
As mentioned in the other comment, always overriding the path name with `/authorize` will break any custom supplied path and we'd need to use `https://id.ai/authorize` explicitly by default since the old II domains don't support ICRC-29.

```suggestion
const IDENTITY_PROVIDER_DEFAULT = 'https://identity.internetcomputer.org';
const IDENTITY_PROVIDER_ICRC29_DEFAULT = 'https://id.ai/authorize;
const IDENTITY_PROVIDER_ENDPOINT = '#authorize';
```

Reply:
Good point